pkg/trace/api/api.go

```diff
@@ -643,13 +643,27 @@ func (r *HTTPReceiver) handleTraces(v Version, w http.ResponseWriter, req *http.
 	payload := &Payload{
 		Source:                 ts,
 		TracerPayload:          tp,
-		ClientComputedTopLevel: req.Header.Get(header.ComputedTopLevel) != "",
-		ClientComputedStats:    req.Header.Get(header.ComputedStats) != "",
+		ClientComputedTopLevel: isHeaderTrue(header.ComputedTopLevel, req.Header.Get(header.ComputedTopLevel)),
+		ClientComputedStats:    isHeaderTrue(header.ComputedStats, req.Header.Get(header.ComputedStats)),
 		ClientDroppedP0s:       droppedTracesFromHeader(req.Header, ts),
 	}
 	r.out <- payload
 }
 
+// isHeaderTrue returns true if value is non-empty and not a "false"-like value as defined by strconv.ParseBool
+// e.g. (0, f, F, FALSE, False, false) will be considered false while all other values will be true.
+func isHeaderTrue(key, value string) bool {
+	if len(value) == 0 {
+		return false
+	}
```

Comment:
This will essentially treat the empty value `""` as `false` as well. It may result in unexpected behavioral changes if the tracer is not setting any value in the header 🤔

Reply:
Yeah this is intentional as the previous behavior also considered the empty value as false, and if the header isn't set at all then it was also considered false (and should continue to be false I think)